### PR TITLE
WWW-384 Play icon on video players floating separately from buttons on Edge v42 (Pega Edge)

### PR DESCRIPTION
**Pull request by colbytcook**
## Jira

https://pegadigitalit.atlassian.net/browse/WWW-384

## Summary

Create Edge ~v.42 specific CSS to handle the video player "play" icon

## Details

Wrapped the "play" icon CSS in @support to serve up two different CSS blocks depending on if the browser supports clip-path

## How to test

View "/pattern-lab/?p=components-video" in Edge v.18 Browserstack (this browser also doesn't support clip-path)

### CODE REVIEW

File: packages/components/bolt-video/src/_edge-video-updates.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  .vjs-big-play-button {
+    .vjs-icon-placeholder {
+      @supports (
+        not
+          (
+            (-webkit-clip-path: polygon(0 0, 0 100%, 100% 50%)) or
+              (clip-path: polygon(0 0, 0 100%, 100% 50%))
+          )
+      ) {
+        top: 40% !important;
```
Comment (adamszalapski) [inline packages/components/bolt-video/src/_edge-video-updates.scss:11]: @colbytcook we really need `!important` here I try to remove it and it works as well.

File: packages/components/bolt-video/src/_edge-video-updates.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        left: 40% !important;
```
Comment (adamszalapski) [inline packages/components/bolt-video/src/_edge-video-updates.scss:12]: @colbytcook we really need `!important` here I try to remove it and it works as well.

**Review by adamszalapski — APPROVED** [review]
It works perfectly. I add two comments in the code.

File: packages/components/bolt-video/src/_edge-video-updates.scss (hunk 1 above)
Comment (colbytcook) [reply packages/components/bolt-video/src/_edge-video-updates.scss:11]: ah ok, let me test that, i was having trouble at one point

Comment (colbytcook) [reply packages/components/bolt-video/src/_edge-video-updates.scss:11]: Edge inspector is strange, the 40% CSS declaration is being overwritten (according to the inspector anyways) but it is not changing the display at all. This is why I put the !important in originally. Well I am removing it now anyways.

File: packages/components/bolt-video/src/_edge-video-updates.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
+.video-js {
```
Comment (mikemai2awesome) [inline packages/components/bolt-video/src/_edge-video-updates.scss:1]: Let's not create a separate file for this. it's really hard to know there are hacks. keep it in the same file and write a comment.

**Review by mikemai2awesome — CHANGES_REQUESTED** [review]

**Review by mikemai2awesome — APPROVED** [review]

**Pull request merged**